plugins/automox/plugin.spec.yaml

```diff
@@ -0,0 +1,988 @@
+      title: SAML Enabled
+      description: Whether SAML has been enabled for the user
+      type: boolean
+      required: false
+    rbac_roles:
+      title: Roles
+      description: The roles assigned to the user
+      type: "[]object"
+      required: false
+  device:
+    id:
+      title: Device ID
+      description: The device ID
+      type: integer
+      required: true
+    server_group_id:
+      title: Server Group ID
+      description: The server group identifier of the device
+      type: integer
+      required: true 
+    organization_id:
+      title: Organization ID
+      description: The organization identifier of the device
+      type: integer
+      required: true
+    uuid:
+      title: Device UUID
+      description: The device unique identifier
+      type: string
+      required: true
+    name:
+      title: Device Name
+      description: The device name
+      type: string
+      required: true
+  policy:
+    id:
+      title: Policy ID
+      description: The policy identifier
+      type: integer
+      required: true
+    organization_id:
+      title: Organization ID
+      description: The organization identifier of the policy
+      type: integer
+      required: true
+    name:
+      title: Policy Name
+      description: The name of the policy
+      type: string
+      required: false
+    policy_type_name:
+      title: Policy Type Name
+      description: The type of policy
+      type: string
+      required: false
+    configuration:
+      title: Policy Configuration
+      description: The configuration of the policy
+      type: object
+      required: false
+    server_groups:
+      title: Device Groups
+      description: List of device groups assigned policy
+      type: "[]integer"
+      required: false
+  group:
+    id:
+      title: Group ID
+      description: The group identifier
+      type: integer
+      required: true
+    organization_id:
+      title: Organization ID
+      description: The organization identifier of the group
+      type: integer
+      required: true
+    name:
+      title: Group Name
+      description: The name of the group
+      type: string
+      required: false
+    refresh_interval:
+      title: Refresh Interval
+      description: Frequency of device refreshes in minutes
+      type: integer
+      required: false
+    parent_server_group_id:
+      title: Parent Server Group ID
+      description: The identifier of the parent group
+      type: integer
+      required: false
+    ui_color:
+      title: Color
+      description: Automox console highlight color for the group
+      type: string
+      required: false
+    notes:
+      title: Notes
+      description: Notes defined for the group
+      type: string
+      required: false
+    enable_os_auto_update:
+      title: Enable OS Auto Update
+      description: Enable operating system auto update
+      type: boolean
+      required: false
+    server_count:
+      title: Device Count
```

Comment:
Again, maybe a bit confusing to have server_count as key here and Device Count as title. Are servers the same as devices?

Reply:
That is correct. The API in the future will be updated to better reflect but it is accurate that servers/devices are interchangeable. Would you rather have in-action mappings of fields for each of these?

Reply:
@ax-zyoutz it should be fine - the user just sees the title properties.